receiver/influxdbreceiver/config.go

```diff
@@ -23,10 +23,4 @@ import (
 type Config struct {
 	config.ReceiverSettings       `mapstructure:"-"`
 	confighttp.HTTPServerSettings `mapstructure:",squash"`
-
-	// MetricsSchema indicates the metrics schema to emit to line protocol.
-	// Options:
-	// - telegraf-prometheus-v1
-	// - telegraf-prometheus-v2
-	MetricsSchema string `mapstructure:"metrics_schema"`
```

Comment:
I know we are still allowed to make breaking changes, but you could have been nicer to your users and deprecating this option first, leaving a big warning in the logs in case this is being used. And effectively remove it in a couple of releases.